### PR TITLE
Update popup.html

### PR DESCRIPTION
**Pull request by srikar0896**
improved javascript style

### CODE REVIEW

**Comment by Ceilican** [comment]
Thanks!

File: Source/popup.html [hunk 1]
```diff
@@ -1,23 +1,19 @@
-<!doctype html>
+<!DOCTYPE html>
 <html ng-app="PopupApp">
   <head>
     <title>Mind The Word</title>
-
     <link href="assets/css/bootstrap.css" rel="stylesheet">
-
-    <script type="text/javascript" src="assets/js/angular.min.js"></script>
+    <script type="text/javascript" src="assets/js/angular.min.js"></script> //angularjs lib
```
Comment (rohankatyal29) [inline Source/popup.html]: @Ceilican , the //angularjs lib will get displayed in popup. This is wrong syntax for a HTML comment

Comment (Ceilican) [inline Source/popup.html]: Oh, yes. I did not notice that.

**Comment by Ceilican** [comment]
@srikar0896 , could you fix that?

File: Source/popup.html (hunk 1 above)
Comment (rohankatyal29) [inline Source/popup.html]: I am just pushing some code, have made the necessary changes in it

Comment (srikar0896) [inline Source/popup.html]: yea. is used for html comment line.I did'nt notice that.

**Pull request merged**